docs/src/projects/configuration.md

```diff
@@ -105,6 +105,56 @@ response: [0x1, 0x23, 0x4]
 </details>
 <br>
 
+### Global configuration
+
+Global configuration file is usual [`snfoundry.toml`](https://foundry-rs.github.io/starknet-foundry/appendix/snfoundry-toml.html), 
+which is a common storage for configurations to apply to multiple projects across various directories.
+This file is stored in a predefined location and is used to store profiles that can be used from any location on your computer.
+
+#### Interaction between Local and Global profiles
+
+Global config can be overridden by a local config.
+
+If both local and global profiles with the same name are present, local profile will be combined with global profile. That is for any setting defined both in global and local profiles, local setting will be used. For settings not defined in local profile, value from the global profile will be used instead.
+
+This same behavior applies for [default profiles](#default-profile) as well. A local default profile will override a global default profile.
+
+#### Global Configuration File Location
+The global configuration is stored in a specific location depending on the operating system:
+
+- macOS/Linux : The global configuration file is located at `$HOME/.config/starknet-foundry/snfoundry.toml`
+- Windows : The file can be found at `C:\Users\<user>\AppData\Roaming\starknet-foundry\snfoundry.toml`
+
+> 📝 **Note**
+> If missing, global configuration file will be created automatically on running any `sncast` command for the first time.
+
+### Config interaction example
+
+```
+root/
+├── .config/
+│   └── starknet-foundry/
+│       └── snfoundry.toml -> A
+└── /../../
+        └── projects/
+            ├── snfoundry.toml -> B
+            └── cairo-projects/
+                └── opus-magnum/
+```
+
+**Glossary:**
+
+- **A:** Global configuration file containing the profiles [`default`](#default-profile) and [`testnet`](#defining-profiles-in-snfoundrytoml).
+- **B:** Local configuration file containing the profiles `default` and `mainnet`.
+
+In any directory in the file system, a user can run the `sncast` command using the `default` and `testnet` profiles, 
+because they are located in global config. 
```

Comment:
```suggestion
In any directory in the file system, a user can run the `sncast` command using the `default` and `testnet` profiles, 
because they are defined in global config (file A). 

```